media-capabilities/media-capabilities.test.js

```diff
@@ -16,7 +16,7 @@
 
```

Comment:
@wingleung cc @addyosmani 

Just in the scope of naming I'd like to suggest updating like the following.